guides/hidden-pages.mdx

```diff
@@ -0,0 +1,32 @@
+---
+title: "How to add hidden pages"
+description: "Exclude pages from your docs.json"
+---
+
+Sometimes you may want to write a page and not publish it immediately.
+
+For content that is not strictly confidential, but isn't ready for prime time, you can add pages to your docs without referencing them in your `docs.json`. This is what is considered a "hidden page".
+
+As long as the page is not described under any "group", "tab" or other element in the navigation section of your `docs.json`, the only way a user will be able to navigate to the page is if they know or guess the page URL.
+
+<Note>
+Some navigation elements like sidebars and tabs may shift or appear empty on hidden pages
+</Note>
+
+If you want to take further steps to hide hidden pages' content, make sure to add `noindex: true` to the frontmatter of your page. This should prevent the page from appearing in external search results. On the other hand, if you do want hidden pages indexed for SEO purposes you can add the following setting to your `docs.json`:
+
+```
+"seo" {
+    "indexing": all
+}
+```
+
+Learn more about SEO and Sitemaps configuration in the [SEO](https://mintlify.com/docs/settings/seo#sitemaps) docs.
+
+<Warning>
+Contents of hidden pages will not be hidden from internal docs search, and may appear in AI-generated results
+</Warning>
```

Comment:
not sure if this is still true based on Slack conversations

Reply:
discussed with Sam and Mayank, the SEO setting applies to the internal search/AI search as well. updated in [95bde84](https://github.com/mintlify/docs/pull/655/commits/95bde84be11af568ba1167b9197417f81e45c466)